### DISCUSSION TITLE
Welcome to moko-socket-io Discussions!

### DISCUSSION THREAD

**Discussion by Alex009** [icerockdev/moko-socket-io#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by viicentecs** [comment]
You can use the Moko socket io, with a path, I just need to send my url without the /socket.io/ how could I do it
